drivers/interrupt_controller/exti_stm32.c

```diff
@@ -521,6 +559,59 @@ static void __stm32_exti_connect_irqs(struct device *dev)
 		CONFIG_EXTI_STM32_RTC_WKUP_IRQ_PRI,
 		__stm32_exti_isr_22, DEVICE_GET(exti_stm32),
 		0);
++#elif CONFIG_SOC_SERIES_STM32F7X
```

Comment:
The shippable build failed due to this leftover from the rebase. Same issue a few lines below.

Reply:
Thank you. I have update the code.